cwl/src/test/scala/wdl4s/cwl/ThreeStepSpec.scala

```diff
@@ -1,51 +1,173 @@
 package wdl4s.cwl
 
-import org.scalatest.FlatSpec
-/*
-import shapeless.Coproduct
-import shapeless.syntax.inject._
-import shapeless.ops.coproduct.Inject
-import mouse.all._
-*/
-
-class ThreeStepSpec extends FlatSpec {
-
-  "Cwl Three step" should "convert to Wom" in {
-
-    //val ps = new CommandLineTool()
-
-    /*
-    val patternInput =
-      InputParameter(
-        Some("pattern"),
-        `type` = Some(Coproduct[MyriadInputType](CwlType.String)))
-    val _inputs = Coproduct[WorkflowInput](Array(patternInput))
-
-    val outputCgrep =
-      WorkflowOutputParameter(
-        id = Some("cgrep.count"),
-        `type` = Some(Coproduct[MyriadOutputType](CwlType.Int)))
-
-    val outputWc =
-      WorkflowOutputParameter(
-        id = Some("wc.count"),
-        `type` = Some(Coproduct[MyriadOutputType](CwlType.Int)))
-
-    //val _outputs = Coproduct[WorkflowInput](Array(outputCgrep, outputWc))
-
-    val psWfStep  = WorkflowStep(
-      id = Some("ps"),
-      inputs = Coproduct[CommandLineTool.Inputs](Array.empty),
+import shapeless._
+import syntax.singleton._
+import wdl4s.cwl.CommandLineTool.{Argument, BaseCommand, Inputs, StringOrExpression}
+import wdl4s.cwl.CommandOutputBinding.Glob
+import wdl4s.cwl.WorkflowStep.{Outputs, Run}
+import io.circe.syntax._
+import io.circe.yaml._
+import io.circe.yaml.syntax._
+import org.scalatest._
+
+class ThreeStepSpec extends FlatSpec with Matchers {
+  val namespace = "threestep"
+  it should "print nicely " in {
+
+    val inlineJScriptRequirements = Option(Array(
+      Coproduct[Requirement](ShellCommandRequirement()),
+      Coproduct[Requirement](InlineJavascriptRequirement())))
+
+  /**
+    * PS
```

Comment:
It took me a lot longer than it should to realize what 'Ps' meant here. Not sure one should cater comments to my mental deficiencies, but .... :)